.github/workflows/prod-deploy.yml

```diff
@@ -1,67 +1,136 @@
+name: Frontend Production Release
+
 on:
   workflow_dispatch:
+    inputs:
+      skip-tagging:
+        type: boolean
+        description: Skip merging and tagging a release.
```

Comment:
Is this needed?

Reply:
I believe the cause for why we ran into issues with needing to skip tagging has been addressed as a part of this PR. Do you think we could still run into a similar situation?

Reply:
I believe it won't solve for it, because we are querying for `label:"autorelease: pending"` in the initial step. But, if it fails on deployment, it wouldn't have that label. It would have the label `autorelease: tagged`.